src/routes/threads/helpers.ts

```diff
@@ -20,6 +20,14 @@ type FilterThreadsArgs = {
     allTags?: boolean;
 };
 
+export function removeBackticks(rawContent: string, maxLength: number = 200): string {
```

Comment:
lets try to find a better name for this function, because it just not only replaces backticks.

Something like sanitizeContent or so 👍🏻

Reply:
Done.